### PR TITLE
Admin Verb to spawn an "ERT Shuttle"

### PR DESCRIPTION
**Pull request by X0-11**
Adds an admin verb to spawn an ERT shuttle that is invisible to all but ghosts and contains many different items (human: vendors of all /armory types, Covvie: many crates of gear).

:cl: XO-11
tweak: Modifies the way lighting overlays are generated for NPC ships, potentically fixing the darkness bug.
/:cl:

### CODE REVIEW

**Review by bloxgate — COMMENTED** [review]
Just to check, does this have any integration with the existing ERT verb?

Will spawning this finally allow us to actually call the ERT?

**Review by BDpuffy420 — APPROVED** [review]

File: code/modules/halo/overmap/base_npc_ships.dm [hunk 1]
```diff
@@ -21,6 +21,8 @@
 
 #define ICON_FILES_PICKFROM list('code/modules/halo/overmap/freighter.dmi','code/modules/halo/icons/overmap/large_cargo_ship.dmi','code/modules/halo/icons/overmap/medical_ship.dmi','code/modules/halo/icons/overmap/mariner-class.dmi','code/modules/halo/icons/overmap/heavy_freighter.dmi')
 
+#define LIGHTRANGE_LIKELY_UNUSED 99
```
Comment (bloxgate) [inline code/modules/halo/overmap/base_npc_ships.dm:24]: Missing an undefine?

**Review by bloxgate — DISMISSED** [review]

File: code/modules/halo/overmap/base_npc_ships.dm (hunk 1 above)
Comment (X0-11) [reply code/modules/halo/overmap/base_npc_ships.dm:24]: whoops

**Comment by X0-11** [comment]
No, it's not integrated with any other admin call ERT verb I'm afraid. Standalone verb in the secrets panel.

**Pull request merged**